c10/core/TensorImpl.h

```diff
@@ -1668,7 +1673,9 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
   void release_storage_and_set_meta_custom_data_ptr_error_msg_(
       c10::optional<std::string> s) {
     storage_ = {};
-    get_extra_meta().custom_data_ptr_error_msg_ = std::move(s);
+    set_storage_access_should_throw();
+    get_extra_meta().custom_data_ptr_error_msg_ = s;
+    get_extra_meta().custom_storage_error_msg_ = s;
```

Comment:
Are CUDA graphs the only user of `release_storage_and_set_meta_custom_data_ptr_error_msg_`?

Reply:
Looks like it to me. If it wasn't, then we may have a BC problem on our hands